backend/src/test/kotlin/org/loculus/backend/SpringBootTestWithoutDatabase.kt

```diff
@@ -24,5 +25,11 @@ annotation class SpringBootTestWithoutDatabase
 class TestConfig {
     @Bean
     @Primary
-    fun dataSource(): DataSource = mockk()
+    fun dataSource(): DataSource = mockk<DataSource>()
+        .also {
+            // fix for: https://github.com/loculus-project/loculus/pull/3064
```

Comment:
I think I would rather link to https://github.com/spring-projects/spring-boot/issues/42323

Reply:
sure